Gopkg.toml

```diff
@@ -32,3 +32,7 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  name = "github.com/davecgh/go-spew"
```

Comment:
Looks like this is something you only use to debug when coding, I dont think we should have this here unless we are including it code somewhere i didnt find, should also remove commented out lines of this.

Reply:
I would agree... I talked about this with @bartlettc22 and in development, we use this a lot. So the idea here is that it is a pain on the bus to have to pull this one down. So for now, we want to leave this one in there. At some point, we might also upgrade this project not to use the older dep... Maybe this will help out with this vendor folder thing as well.